sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/CreateHiveTableAsSelectCommand.scala

```diff
@@ -83,10 +84,12 @@ trait CreateHiveTableAsSelectBase extends DataWritingCommand {
     tableDesc: CatalogTable,
     tableExists: Boolean): DataWritingCommand
 
+  def writingCommandClass: Class[_]
+
   override def argString(maxFields: Int): String = {
     s"[Database:${tableDesc.database}, " +
```

Comment:
Looks we need a space between `Database:` and `${tableDesc.database}`.

Reply:
Done, thanks!